08Lesson/01Project/index.html

```diff
@@ -1,10 +1,37 @@
 <!DOCTYPE html>
 <html>
 <head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css">
+  <title> My Media Query </title>
+  <meta charset="utf-8">
+  <link rel="stylesheet" href="css/reset.css">
+  <link rel="stylesheet" media="(min-width: 768px)" href="css/style.css">
+  <link rel="stylesheet" href="css/phone.css" media="(max-width: 767px)">
```

Comment:
This is an interesting and very well-organized approach to linking your styles. It's a good practice to make your codebase modular when you can - I'm pleased that you're already thinking that way!

There's an issue with your `phone` media query - the `<h1>` is losing its height definition whenever the viewport shrinks below `767 px`.  Should be a pretty easy fix. Other than that, nice job!